client/interfaces/IPageStateDefinitions.ts

```diff
@@ -0,0 +1,18 @@
+export default interface IPageStateDefinitions {
+  [state: string]: IPageStateDefinitionFn;
+}
+
+export type IPageStateDefinitionFn = (assert: IPageStateAssertionFns) => void;
```

Comment:
Should the Fn's argument still be called `assert`? Shouldn't it be assertionFns?